### ISSUE TITLE
[FEATURE] Convert Compositions to Crossplane Functions

### ISSUE THREAD

**Issue by blakeromano** [awslabs/crossplane-on-eks#159]
#### Is your feature request related to a problem? Please describe
With 1.14 Release of Crossplane Release and the opensource P&T function we should look to convert the Crossplane on EKS Compositions to use the function. There is a conversion CLI here https://github.com/crossplane-contrib/crossplane-migrator that can be used to help automate this process.

#### Describe the solution you'd like
I'd like to see the conversion to P&T function be done and see compositions usable with composition function pipelines.

**Comment by nabuskey** [comment]
Agreed. Since the feature is in beta now, we should definitely consider doing this. @candonov ^^.

**Comment by candonov** [comment]
Absolutely 💯! @blakeromano, are you interested in contributing?

**Comment by blakeromano** [comment]
I am not sure I have the time to do all the compositions. I did a quick example here https://github.com/blakeromano/crossplane-on-eks/tree/feature/start-pt-function-move if I could get help with testing it that'd be super appreciated.

**Comment by huytn1219** [comment]
I can help contributing to this issue. @candonov

**Comment by nabuskey** [comment]
Since we have many compositions, we should start with a simple ones like the SQS and S3 ones.

**Comment by candonov** [comment]
Thank you @huytn1219. This [s3](https://github.com/awslabs/crossplane-on-eks/tree/main/compositions/upbound-aws-provider/s3) composition would be a good one to start with.
